cmd/containerd-shim-runhcs-v1/delete.go

```diff
@@ -82,6 +83,13 @@ The delete command will be executed in the container's bundle as its cwd.
 			}
 		}
 
+		// hcsshim shim writes panic logs in the bundle directory in a file named "panic.log"
+		// log those messages (if any) on stderr so that it shows up in containerd's log.
+		logs, err := ioutil.ReadFile(filepath.Join(bundleFlag, "panic.log"))
+		if err == nil && len(logs) > 0 {
+			fmt.Fprintf(os.Stderr, "shim panic: %s", string(logs))
```

Comment:
Can we log this via logrus instead? Then we will see it as an event from the runhcs ETW provider.

Reply:
I would also make the message a little clearer, maybe "found shim panic log during delete".

Reply:
We can but this entire file specifically avoids writing anything with logrus. At the very top it sets the logrus output to `ioutil.Discard` in order to avoid writing anything to stdout and then writes all messages to stderr (from where containerd picks it up). However, the default output of logrus is `stderr` not `stdout` so we can just logrus everywhere. But for now I will remove that `logrus.SetOutput(ioutil.Discard)` statement at top and write this message with logrus.

Reply:
Actually we could even leave in the `logrus.SetOutput` and it should still be okay, since we only care about the logrus hook (ETW) anyways.

Reply:
But then we won't see the panic logs in containerd event logs so it won't really help unless we are capturing an ETW trace.